requirements/pip_tools.in

```diff
@@ -0,0 +1,2 @@
+# Dependencies to run compile tools
+pip-tools                           # Contains pip-compile, used to generate pip requirements files
```

Comment:
You probably don't need it yet, but it's a good idea to have a `constraints.txt` file when something needs to be prevented from upgrading for a while, referenced in each input requirements file: https://open-edx-proposals.readthedocs.io/en/latest/oep-0018-bp-python-dependencies.html#imposing-constraints-on-dependencies

Reply:
Yeah, I wasn't sure if should add an empty file or not, but I can, just so that the `-c` call is in each `.in` file

Reply:
There's an example with an explanatory comment at https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/requirements/constraints.txt